src/components/CardFront.tsx

```
@@ -264,17 +264,35 @@ const CharacterPlays = ({
+                    </div>,
+                  CP2:
+                    <div style={{
+                      width: "1em", height: "1cap",
+                      display: "inline-flex",
+                      alignItems: "flex-end",
+                      justifyContent: "flex-start",
+                    }}>
+                      <GBIcon icon={gbcp ? "trophy" : "GBT"} size='1.2cap' />
+                    </div>
+                }[s] || s}
+              </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)